pom.xml

```diff
@@ -1333,6 +1353,12 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
```

Comment:
Where is the dependency used? Can it be replaced with the `io.dropwizard.metrics`?

Reply:
I am not sure why is it necessary to have `com.yammer.metrics` in dependencyManagement?

Reply:
I meant that it can be useful if some new dependencies will have older transitive `org.yummer.metrics` in future. 
But I do not mind, this isn't so important. Let's leave it without dependencyManagement control.